docker-compose.override.yml

```diff
@@ -0,0 +1,32 @@
+version: '3'
+services:
+  mongo:
+    image: mongo:3.4
+    ports:
+    - "27017:27017"
+  rabbit:
```

Comment:
-1 from me. When I need to exec commands on a service like mongo, I like the service having the name mongo for exampl.e

Reply:
@warrenv I'm also more comfortable with leaving them as they are, implementation-specific, since 99% of the way I treat them depends on the implementation... 

The application consuming them doesn't really mind what they are called, it only needs the URL, but to me eye it helps me know exactly what I'm dealing with when doing manual stuff.

Reply:
:+1: